### PR TITLE
Tilt monitor

### PR DESCRIPTION
**Pull request by Matthew-En**

### CODE REVIEW

File: src/org/usfirst/frc/team467/robot/TiltMonitor.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc.team467.robot;
+
+import org.apache.log4j.Logger;
+
+public class TiltMonitor {
```
Comment (john-petrangelo) [inline src/org/usfirst/frc/team467/robot/TiltMonitor.java]: You should write a class comment describing what this class is for.

File: src/org/usfirst/frc/team467/robot/TiltMonitor.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+	public static TiltMonitor getInstance() {
+		if (instance == null) {
+			instance = new TiltMonitor();
+		}
+		return instance;
+	}
+	public void periodic() {
+		if (Math.abs(Gyrometer.getInstance().getPitchDegrees()) > 45) {
```
Comment (john-petrangelo) [inline src/org/usfirst/frc/team467/robot/TiltMonitor.java]: You should use separate numbers for forward and backwards tilt - the robot may be more tippy forward or backward.

You should put move angles into robot map. Call them FORWARD_PANIC_ANGLE and BACKWARD_PANIC_ANGLE. You can set them both to 45 to start.

File: src/org/usfirst/frc/team467/robot/TiltMonitor.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+			LOGGER.info("Pitch=" + Gyrometer.getInstance().getPitchDegrees());
```
Comment (john-petrangelo) [inline src/org/usfirst/frc/team467/robot/TiltMonitor.java]: Change this log message from info to debug.

File: src/org/usfirst/frc/team467/robot/TiltMonitor.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+			Elevator.getInstance().moveToHeight(Elevator.Stops.floor);
+
```
Comment (john-petrangelo) [inline src/org/usfirst/frc/team467/robot/TiltMonitor.java]: Remove extra blank line.

**Review by john-petrangelo — CHANGES_REQUESTED** [review]

File: src/org/usfirst/frc/team467/robot/TiltMonitor.java (hunk 1 above)
Comment (john-petrangelo) [reply src/org/usfirst/frc/team467/robot/TiltMonitor.java]: I'll help you with the format for this comment.

File: src/org/usfirst/frc/team467/robot/TiltMonitor.java [hunk 5]
```diff
@@ -1,23 +1,27 @@
...
+*/
 public class TiltMonitor {
 	private static TiltMonitor instance;
-	private static final Logger LOGGER = Logger.getLogger(TiltMonitor.class);    
-
-
+	private static final Logger LOGGER = Logger.getLogger(TiltMonitor.class);
+	
+	public static final int FORWARD_PANIC_ANGLE = 45;
+	public static final int BACKWARD_PANIC_ANGLE = -45;
```
Comment (john-petrangelo) [inline src/org/usfirst/frc/team467/robot/TiltMonitor.java]: These should go into RobotMap.

**Review by john-petrangelo — CHANGES_REQUESTED** [review]

**Comment by Matthew-En** [comment]
All comments fixed were done by Hasan and me.

**Review by john-petrangelo — APPROVED** [review]

**Pull request merged**